packages/command/test/unit/sfdxFlags.test.ts

```diff
@@ -353,6 +353,18 @@ describe('SfdxFlags', () => {
           'The flag value "1,2,c" is not in the correct format for "array." Must only contain valid values.'
         );
       });
+
+      it('should handle various arrangments of comma separated lists without errors', () => {
+        const array = flags.array({ description: 'test' });
+        if (!hasFunction(array, 'parse')) throw new MissingPropertyError('parse', 'array');
+        expect(array.parse('1,2,3,4,5,6')).to.deep.equal(['1', '2', '3', '4', '5', '6']);
```

Comment:
I think this one is actually a problem. If a user passes in '1,2,3,4,5,6' we shouldn't be splitting those up, we should just return ['1,2,3,4,5,6']. 

@mdonnalley